src/main/resources/default/assets/tycho/scripts/tooltips.js

```diff
@@ -0,0 +1,5 @@
+// Initializes all Bootstrap tooltips...
+sirius.ready(function() {
+    // This needs to use jQuery for Bootstrap compatibility reasons...
```

Comment:
Optional: Thats why it would be nice to upgrade to BS 5.0.1 right away as it doesnt seem to require jQuery for things like this: https://getbootstrap.com/docs/5.0/components/tooltips/

Reply:
Yep, but I think moving to BS5 requires its own PR and approach without any modifications next to it.